HARSHAL/javascript/assignment10/assignment10Q1.js

```diff
@@ -0,0 +1,8 @@
+var a = prompt("write one word");
+var firstindex = a.charAt(0);
```

Comment:
Use Regular Expression: Checking string is start with upperCase or not
/^[A-Z]/

Reply:
yes use the regular expression for checking